docs/controls/file-pickers/node/select-files.md

```diff
@@ -0,0 +1,102 @@
+---
+author: keco
+ms.author: keco
+ms.date: 10/02/2018
+---
+# Selecting Files with the Microsoft File Browser SDK
+
+The following walk through shows how to integrate the [Microsoft File Browser SDK](https://www.npmjs.com/package/@microsoft/file-browser) with the React application we created during [project setup](index.md).
+
+### Select Files Demo
```

Comment:
It looks like the ordering of the header sizes are swapped here. In index.md you have ## for the Setup header and ### for all the others, which I think would look consistent to have here as well. So, maybe ## Select Files Demo and ### 1. Pass ... ### 2. Attach, and so on.